### ISSUE TITLE
[Bug] PaddlePaddle integration tests busted at main

### ISSUE THREAD

**Issue by areusch** [apache/tvm#9231]
We temporarily had a problem with our post-merge CI builds, so apologies this wasn't caught quite as quickly as we normally would. Seems that 5 integration tests related to PaddlePaddle are busted at main likely due to a merge conflict.

https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/main/1883/pipeline

FAILED tests/python/driver/tvmc/test_compiler.py::test_compile_paddle_module
FAILED tests/python/driver/tvmc/test_compiler.py::test_cross_compile_aarch64_paddle_module
FAILED tests/python/driver/tvmc/test_compiler.py::test_cross_compile_options_aarch64_paddle_module
FAILED tests/python/driver/tvmc/test_frontends.py::test_load_model__paddle - ...
FAILED tests/python/driver/tvmc/test_tvmc_common.py::test_compile_paddle_module_nchw_to_nhwc

I suspect #9083 might be related. Is anyone more familiar with it able to look? cc @jiangjiajun @masahi @leandron

**Comment by masahi** [comment]
Yeah, #9083 was merged three days ago and as far as I know, there have been no problem with that tests until yesterday. My hunch is that #9126, which apparently didn't run CI job after #9083, broke it.

**Comment by masahi** [comment]
https://github.com/apache/tvm/pull/9232 is looking good

**Comment by jiangjiajun** [comment]
I have reproduced the problem, and send a pull request to fix the bug, let's see if this could solve the ci problem @masahi @junrushao1994 @areusch

**Comment by junrushao1994** [comment]
Thanks @jiangjiajun for the quick response! I merged the hotfix PR just now. Let's see what will happen on mainline
